packages/oauth/test/unit/lib/authorizations.unit.test.ts

```diff
@@ -0,0 +1,93 @@
+import * as Heroku from '@heroku-cli/schema'
+import {expect, test} from '@oclif/test'
+import * as addSeconds from 'date-fns/add_seconds'
+import * as distanceInWordsToNow from 'date-fns/distance_in_words_to_now'
+
+import {display} from '../../../src/lib/authorizations'
+
+const setupDisplay = (auth: Heroku.OAuthAuthorization) =>
+  test
+    .stdout()
+    .do(() => display(auth))
+
+describe('display', () => {
+  context('with an auth', () => {
+    const auth: Heroku.OAuthAuthorization = {
+      id: 'f6e8d969-129f-42d2-854b-c2eca9d5a42e',
+      description: 'a cool auth',
+      scope: ['global', 'app'],
+    }
+
+    setupDisplay(auth)
+      .it('prints the styled authorization', ctx => {
+        expect(ctx.stdout).to.contain('ID:          f6e8d969-129f-42d2-854b-c2eca9d5a42e\n')
```

Comment:
Can we turn the id's in this file into a variables and re-use?
`6e8d969-129f-42d2-854b-c2eca9d5a42e`

Reply:
Done 👍